manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
+PackageVersion: 2.9.3.4

```

```suggestion
PackageVersion: 2.9.3.4
Commands:
- ln
```
To be able to call the executable through command-line by `ln`